collections.go

```diff
@@ -77,8 +77,10 @@ type CollectionDTO struct {
 	ID             int64     `json:"id"`
 	PublishedAt    time.Time `json:"published_at"`
 	PublishedScope string    `json:"published_scope"`
+	Rules          RuleDTOs  `json:"rules"`
 	SortOrder      string    `json:"sort_order"`
 	TemplateSuffix string    `json:"template_suffix"`
+	ProductCount   int       `json:"product_count"`
```

Comment:
I can't see this property in the docs?

Reply:
yeah I didn't but if you look at the calls you are able to retrieve this field. again for some reason it is not in the docs